moveit_ros/moveit_servo/src/utils/command.cpp

```diff
@@ -122,6 +129,13 @@ JointDeltaResult jointDeltaFromJointJog(const JointJogCommand& command, const mo
   else
   {
     status = StatusCode::INVALID;
+    if (!command_valid)
+    {
+      RCLCPP_WARN_STREAM(getLogger(), "Invalid joint jog command. Each joint name must have one corresponding "
```

Comment:
Should this be an ERROR instead?

Reply:
Also again, if you move these warnings to the place they happen when you actually set `command/names_valid`, you don't need more if statements here